scipy/sparse/linalg/matfuncs.py

```diff
@@ -592,6 +592,15 @@ def _expm(A, use_exact_onenorm):
     if len(A.shape) != 2 or A.shape[0] != A.shape[1]:
         raise ValueError('expected a square matrix')
 
+    # Trivial case
+    if A.shape == (1, 1):
+        data = np.array([[A.data[0]]])
```

Comment:
This will fail now if `A` is an ndarray.  You know the shape is `(1, 1)` so just do `A[0, 0]` to retrieve the value.